liquibase-core/src/main/java/liquibase/change/core/RawSQLChange.java

```diff
@@ -53,7 +35,8 @@ public String getSql() {
         return super.getSql();
     }
 
-    @DatabaseChangeProperty(serializationType = SerializationType.NESTED_OBJECT, exampleValue = "What about Bob?")
+    @DatabaseChangeProperty(serializationType = SerializationType.NESTED_OBJECT, description = "A brief descriptive comment",
```

Comment:
@nvoxland, is this comment written to the database? If so, Adrian, we should update this description to say so.

Reply:
I'm 92.5%  😁  sure most of the inline comments in a raw SQL are not stored in the DB.